### ISSUE TITLE
Crear la tabla "CamaraAplicacionesUsuarios"

### ISSUE THREAD

**Issue by ronaldfabra** [CamComercioSM/apps#81]
Se debe crear la tabla "CamaraAplicacionesUsuarios" para que puedan ser almacenados los datos de los usuarios que se logean o inician sesión dentro de la tienda de apps.
Dentro de esta tabla deben estar almacenados las siguientes columnas:

- usuarioID (INT, PK, NO NULL, AUTO-INCREMENTABLE)
- usuarioGoogleID (DECIMAL, NO NULL, UNIQUE)
- usuarioGoogleEmail (VARCHAR(255), NO NULL)

**Comment by ronaldfabra** [comment]
Se debe creó la tabla "CamaraAplicacionesUsuarios" donde serán almacenados los datos de los usuarios que se logean o inician sesión.
Esto se puede ver a continuación:
![image](https://user-images.githubusercontent.com/30238979/29669311-986630c0-88a8-11e7-9cb8-5203d8c109be.png)

**Comment by jpllinasCCSM** [comment]
No.  Estos datos deben ser una extensión de los datos de usuario del SICAM. es decir deben afectar es a la tabla de usuarios del SICAM

**Comment by ronaldfabra** [comment]
Se verificó y se determinó que Para mostrar los datos de el usuario que desea iniciar sesión en la tienda de aplicaciones, primero se debe verificar que el usuario haya dado permisos a la tienda para obtener sus datos, luego se obtiene el email de el usuario,  luego se ejecuta una consulta en la tabla Usuario con el parametro "email" para verificar que exista en la base de datos, y posteriormente si existe ese usuario  se ejecuta una consulta en la tabla Colaboradores con el parametro "usuarioID" que arrojó la consulta anterior para extraer todos los datos de dicha persona de la base de datos, para así poder mostrarlo en la base de datos. en este caso se desea mostrar el correo y el nombre, por lo tanto se crearon dos variables para asignarles sus respectivos parametros y enviarselos a la vista por url, y si no existe el usuario, se le manda un value por url con valor "usuario no registrado".
esto se puede ver a continuación:
![image](https://user-images.githubusercontent.com/30238979/29733552-544d5326-89b3-11e7-80f5-920616b7dcd9.png)
